trading-hello-service/build.sbt

```diff
@@ -0,0 +1,13 @@
+// Nexus configuration
```

Comment:
Are we publishing the service to Nexus as well? I thought only the API would be published.

Reply:
This is important.
Since the service doesn't publish anything if you run `sbt publish` you'll have error for service.
You have to disable publishing for service cempletely.

Reply:
This may be my miscommunication, I had a call with @danryc today where we analyzed the build. I may have said that we want to publish both projects, but you are right - not the service. This one needs a `docker:publish`, and it's a different task.

Reply:
I have removed build.sbt from `tradingHelloService` project and used `sbt "project tradingHelloServiceApi" publish` in CI